src/xrCore/LocatorAPI.cpp

```diff
@@ -661,6 +661,9 @@ bool CLocatorAPI::Recurse(pcstr path)
 #elif defined(LINUX)
     glob_t globbuf;
 
+    while (char* sep = strchr(scanPath, '\\'))
+        *sep = '/';
```

Comment:
Please consider moving this code (slashes conversion) to separate function (named like ConvertGamePathToPlatformSpecificPath) with platform-specific implementation instead of IFDEFing it here directly. Also it allows to reduce code duplication.